rank-info/src/fetcher.rs

```diff
@@ -0,0 +1,260 @@
+use std::sync::{Arc, Mutex};
+
+use slippi_gg_api::APIClient;
+use slippi_user::UserManager;
+
+use super::{Message, RankInfo, RankManager, RankManagerData};

```

Comment:
This is more cosmetic but most of the imports in this repo get grouped something like the following:

```
// system imports, e.g std::{...}
// third-party-crates, e.g serde_json::{...}
// slippi-module imports that aren't in this module, e.g slippi_gg_api::{...}
// anything module specific, e.g super::{...}

(We might be able to configure a lint for this or something but it's nice to have when scanning a codebase visually)
```